docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,42 +1,116 @@
-////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
+//Only enable one of the following attributes at the same time
+// enable this attribute to show downstream markup and content, must be disabled for upstream doc builds
+//:downstream:
+// enable this attribute to show downstream markup and content, must be disabled for downstream doc builds
+:upstream:
```

Comment:
I'd really advocate for `:community:` over `:upstream:` because it is more meaningful to community contributors who might not realize what "upstream" refers to.

Reply:
Please see my comment above, I think I'll go back to using :downstream: only:)